test/correctness/tracing.cpp

```diff
@@ -132,6 +133,13 @@ int main(int argc, char **argv) {
     g.store_root().compute_at(f, x);
     g.trace_stores().trace_loads().trace_realizations();
 
+    input.trace_loads();
+
+    // Make it all-zero so the existing output values are the same
```

Comment:
By "the same" I'm guessing you mean the same as they were before you made this change, but that won't be apparent to anyone reading this later. Maybe just cut the comment.